### ISSUE TITLE
when CDATA  contain html tag

### ISSUE THREAD

**Issue by night-king** [matanshiloah/xml-parser#26]
when CDATA  contain html tag，this parser Failure!!!
xmlParser.js  line#5:
``` javascript
 var cleanXmlText = xmlText.replace(/\s{2,}/g, ' ').replace(/\\t\\n\\r/g, '').replace(/>/g, '>\n');
``` 
replace all ">" to \n
also cantain html end tag.
so what should we do?

**Comment by alde** [comment]
Also seems to be the case if an attribute contains a '>'

I'm trying to parse a Junit XML and a lot of the TestSuites have name attributes such as 
 ```
name="<ReactComponent /> renders without blowing up"
```

**Comment by matanshiloah** [comment]
Fixed both issues in v1.1.5
